### DISCUSSION TITLE
Calculating forces with with evaluateRforces() and evaluatezforces() with Irrgang13I

### DISCUSSION THREAD

**Discussion by edan0314** [jobovy/galpy#618] [Q&A] [answered]
Hello, I need to clarify a doubt.
evaluateRforces() and evaluatezforces() functions are shown in Galpy's documentation with the potential MWPotential2014, which does not have internal units, has an output in Galpy's normalized units, and needs conversion.force_in_kmsMyr() function to convert to the units of km/sMyr. However, in the case where the potential I use is Irrgang13I, which has internal units, is the output also given in normalized units? If it is given in a specific system of units that is not the Galpy's normalized units, how do I convert to the units to km/sMyr?

**Comment by jobovy** [comment, marked as answer]
Hi,

Yes, this is confusing, but you're correct to note that:
* If a Potential is set up _without_ units (like `MWPotential2014`), then values returned are in internal units
* If a Potential is set up _with_ units, then return values have physical units.

The least confusing way to deal with units is to set `astropy-units = False` in the [configuration file](https://docs.galpy.org/en/latest/installation.html#configuration-file); that way all physical return values are `astropy` Quantities with clear units and if you get a float, it's in internal units. 

For `MWPotential2014`, you can also get physical output doing, e.g.
```python
evaluateRforces(MWPotential2014,1.,0.,ro=8.,vo=220.,use_physical=True)
```
or
```python
evaluateRforces(MWPotential2014,1.,0.,ro=8.,vo=220.,use_physical=True,quantity=True)
```
to force Quantity output.

**Comment by edan0314** [comment]
I am using Irrgang13I potential as 'asi', so if I understood, the output is already in another system of units, that is different from the normalized units. I understood the examples that you gave me in this answer with the MWPotential2014. But what should I do when I use Irrgang13I?
`evaluateRforces(asi,R_min_norm,z_min_norm.,ro=8.4,vo=242.,use_physical=True)`
This way the answer will be given in a specific system of units, but which system?
Another question that occurred to me is the value of the argument, does it have to be the normalized values? I mean R_min_norm=R_min/8.4 and z_min_norm is the same but for the z coordinate.

**Reply by jobovy** [reply]
Because `Irrgang13I` is defined in physical units, to be consistent you need to always use their `ro=8.4,vo=242.`. Your `evaluateRforces` line will return a force (really an acceleration or force per unit mass) in units of km/s/Myr (see [the docs](https://docs.galpy.org/en/latest/getting_started.html#units-in-galpy)).

Arguments that are not specified using `astropy` Quantities are assumed to be in internal units, so in your case, yes, if you input a float it should be R_min_kpc/ro etc.
